newIDE/app/src/EventsSheet/EventsTree/Renderers/CommentEvent.js

```diff
@@ -1,83 +1,78 @@
 // @flow
+import { t } from '@lingui/macro';
+
 import * as React from 'react';
-import ReactDOM from 'react-dom';
 import classNames from 'classnames';
-import { type EventRendererProps } from './EventRenderer';
-import { rgbToHex } from '../../../Utils/ColorTransformer';
+import TextField from '../../../UI/TextField';
 import {
   largeSelectedArea,
   largeSelectableArea,
   selectableArea,
+  disabledText,
 } from '../ClassNames';
+import { type EventRendererProps } from './EventRenderer';
 const gd = global.gd;
 
 const styles = {
   container: {
-    minHeight: 30,
     display: 'flex',
-    backgroundColor: '#fbf3d9',
-  },
-  text: {
-    flex: 1,
-    whiteSpace: 'pre-line',
-    margin: 0,
+    flexWrap: 'wrap',
     padding: 5,
+    overflow: 'hidden',
   },
-  textArea: {
-    padding: 5,
-    flex: 1,
-    boxSizing: 'border-box',
+  textField: {
     width: '100%',
     fontSize: 14,
   },
 };
 
-type State = {|
-  editing: boolean,
-  height: number,
-|};
-
 export default class CommentEvent extends React.Component<
   EventRendererProps,
-  State
+  *
 > {
   state = {
     editing: false,
-    height: 0,
+    height: '100%',
```

Comment:
Is this height: 100% any useful? 

Also it seems to be always 100%, so you can just remove it from the state and put "100%" directly in the style where it's needed :)